tools/transform_to_adapt/transform_problems.py

```diff
@@ -302,13 +243,36 @@ def transform_problem():
     adaptive_problem.extend(
         (script, text_question, iframe_text_question, form_questions, customresponse, complete_text)
     )
-    output_file = os.path.join(parameters.output_dir, os.path.split(parameters.xmlfile)[1])
+    output_file = os.path.join(output_dir, os.path.split(problem_file)[1])
     with open(output_file, 'wb') as fhandler:
         fhandler.write(pretty_print(adaptive_problem).encode('utf-8'))
 
 
+def processing_problem_source(params, dir_walking, capa_template):
+    """
+    Processing transformation for chosen file(s) from the problems source
+
+    :param params: script expected parameters
+    :param dir_walking: boolean to mark source is directory
+    """
+    pattern = params.regex or ''
+
+    if not dir_walking:
+        return transform_problem(params.xmlsource, params.output_dir, capa_template)
+
+    root, dirs, files = os.walk(params.xmlsource).next()
+    for problem in files:
+        if re.search(pattern, problem) and problem.endswith('.xml'):
+            print("Transforming {} problem file".format(problem))
+            transform_problem(os.path.join(root, problem), params.output_dir, capa_template)
+
+
 def main():
-    transform_problem()
+    parameters = get_parameters()
+    dir_walking, capa_template = check_problems_source(parameters)
+
+    processing_problem_source(parameters, dir_walking, capa_template)
+    print("Transformation done! Adaptive problems are in the {} dir.".format(parameters.output_dir))
```

Comment:
`Adaptive problems are in the {} dir` -> `Your can find course with converted problems in {} directory.`